pkg/services/ngalert/CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ Scopes must have an order to ensure consistency and ease of search, this helps u
   - `grafana_alerting_ticker_interval_seconds`
 - [ENHANCEMENT] Create folder 'General Alerting' when Grafana starts from the scratch #48866
 - [ENHANCEMENT] Rule changes authorization logic to use UID folder scope instead of ID scope #48970
+- [ENHANCEMENT] Failures at a database-level won't halt rule evaluation #49857
```

Comment:
Technically a BUGFIX but sadly the changes to this PR are buried among refactor commits and I couldn't pinpoint where exactly was the regression.